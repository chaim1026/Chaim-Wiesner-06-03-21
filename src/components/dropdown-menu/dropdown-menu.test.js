import React from 'react';
import { shallow } from 'enzyme';
import DropdownMenu from './dropdown-menu.component';

it('should render DropdownMenu component', () => {
    const mockCurrencies = [
        {
            symbol: 'ILS',
            rate: 3.31
        }
    ]
    expect(shallow(<DropdownMenu currencies={mockCurrencies} />)).toMatchSnapshot();
});