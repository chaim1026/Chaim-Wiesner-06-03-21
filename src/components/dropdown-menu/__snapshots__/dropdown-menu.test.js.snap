// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DropdownMenu component 1`] = `ShallowWrapper {}`;
